sdk/src/client/node_api/core/routes.rs

```diff
@@ -219,7 +219,7 @@ impl ClientInner {
         Ok(BlockId::from_str(&resp.block_id)?)
     }
 
-    /// Finds a block by its BlockId. This method returns the given block object.
+    /// Finds a block by its ID and returns it as object.
```

Comment:
`it as`, do you mean `its` or `it as an`

Reply:
I think `it as object` is correct tbh